source/keyboardHandler.py

```diff
@@ -369,22 +369,17 @@ def _get_mainKeyName(self):
 		return winUser.getKeyNameText(self.scanCode, self.isExtended)
 
 	def _get_modifierNames(self):
-		modTexts = set()
+		modTexts = []
 		for modVk, modExt in self.generalizedModifiers:
 			if isNVDAModifierKey(modVk, modExt):
-				modTexts.add("NVDA")
+				modTexts.append("NVDA")

```

Comment:
It gets converted to lower case when normalised. However, we want it to be displayed in upper case in log output, since that's how we write it in documentation.